src/solvers/dg_tree/containers_2d.jl

```diff
@@ -75,18 +73,21 @@ end
 
 # Create element container and initialize element data
 function init_elements(cell_ids, mesh::TreeMesh2D,
-                       equations::AbstractEquations{2, NVARS},
-                       basis::LobattoLegendreBasis{T, NNODES}, ::Type{RealT}, ::Type{uEltype}) where {RealT<:Real, uEltype<:Real, NVARS, T, NNODES}
+                       equations::AbstractEquations{2},
+                       basis, ::Type{RealT}, ::Type{uEltype}) where {RealT<:Real, uEltype<:Real}
   # Initialize container
   n_elements = length(cell_ids)
-  elements = ElementContainer2D{RealT, uEltype, NVARS, NNODES-1}(n_elements)
+  elements = ElementContainer2D{RealT, uEltype}(
+    n_elements, nvariables(equations), nnodes(basis))
 
-  init_elements!(elements, cell_ids, mesh, basis.nodes)
+  init_elements!(elements, cell_ids, mesh, basis)
   return elements
 end
 
-function init_elements!(elements, cell_ids, mesh::TreeMesh2D, nodes)
-  n_nodes = length(nodes)
+function init_elements!(elements, cell_ids, mesh::TreeMesh2D, basis)
+  nodes = get_nodes(basis)
+  reference_length = integrate(one ∘ eltype, nodes, basis)
```

Comment:
I know I know... But to anyone not with a strong Julia background this operator is non-self-explanatory and I wouldn't even know how to search for it, since I don't know the name of it (except copy paste).

Maybe change
`ith constant value unity`
to `with constant value unity on the corresponding element data type (using \circ)`

That would make me very happy 😊 😉

Reply:
Done in 79cbb88b

Reply:
❤️